bezier-rs/lib/src/lib.rs

```diff
@@ -199,11 +201,9 @@ impl Bezier {
 		}
 	}
 
-	/// Calculate the point on the curve based on the `t`-value provided.
-	/// Basis code based off of pseudocode found here: <https://pomax.github.io/bezierinfo/#explanation>.
-	pub fn compute(&self, t: f64) -> DVec2 {
-		assert!((0.0..=1.0).contains(&t));
-
+	///  Calculate the point on the curve based on the `t`-value provided.
+	///  Basis code based off of pseudocode found here: <https://pomax.github.io/bezierinfo/#explanation>.
```

Comment:
Please revert this change which added another space after the `///` (unless there's a reason for it?).

Reply:
Not sure how that happened, but thanks for catching it!